src/styles/index.scss

```diff
@@ -1,42 +1,44 @@
+@import 'variables';
```

Comment:
Not sure if this is really needed w/ the single css file (or even the right pattern), but thought I would pull it out into a separate file as opposed to throwing a bunch of variables at the top of the file. 🤷‍♂️